docs/guides/sdk/getting-started.md

```diff
@@ -0,0 +1,226 @@
+---
+contributors: Esther Agbaje
+description: Learn how to use the new Directus JavaScript SDK to access your data.
+---
+
+# Getting Started with the JavaScript SDK
+
+Learn how to use the new Directus JavaScript SDK to easily access and manipulate your data.
+
+## Features
+
+This SDK is a significant overhaul of the previous version, introducing new features and improvements.
+
+- **TypeScript first:** Written in TypeScript, the SDK provides a robust and type-safe development experience
+- **Modular architecture:** The SDK is split into separate modules, giving you granular control over which features to
+  include and which can be pruned at build-time
+- **Lightweight and dependency-free:** It does not require external libraries, ensuring a lighter bundle and streamlined
+  experience.
+
+::: warning SDK in Beta
+
+The new SDK is in Beta, so we recommend using it in a staging environment first before deploying it to production.
+
+:::
+
+## Installation
+
+Install the SDK using the following command:
+
+```js
+npm install @directus/sdk@beta
+```
+
+## Basic Usage
+
+Create the directus client using the `useDirectus` hook from `@directus/sdk@beta`. The client gives you access to your
+Directus project and data.
+
+```ts
+import { useDirectus } from '@directus/sdk@beta';
+
+// your Directus collection schema
+type Schema = {
+	// ...
+};
+
+const client = useDirectus<Schema>('http://directus.example.com');
+```
+
+## Creating a Composable Client
+
+The new SDK introduces the concept of a "Composable Client" which allows you to customize and build a client with the
+specific features you need.
+
+The client starts as an empty wrapper without any functionality. To add features, use the following composables:
+
+- `rest()`: REST request functions, adds `.request(...)` to the client.
+- `graphql()`: GraphQL request functions, adds `.query(...)` to the client.
+- `auth()`: Authentication functions, adds `.login(...)`, `.logout()`, and .`refresh()` to the client. (work in
+  progress).
+- `realtime()`: WebSocket connectivity, adds `.subscribe(...)` and `.message(...)` to the client. (work in progress).
+- `subscription()`: GraphQL Subscriptions, adds `.subscription()` to the client. (work in progress).
```

Comment:
@estheragbaje @br41nslug can we add a note on what 'Work in progress' means here? Does it work... at all? Is it indicative of coming soon functionality? Need to set expectations clearly.

Reply:
The parts marked work in progress are planned but either still under development or todo and very likely dont work yet. i have updated the readme and getting started.